### PR TITLE
Create view to edit owner information

### LINKED ISSUE #72: Create View to edit pet owner information.
## Description
Create View to edit pet owner information.

### User story:
User can see previous entered information and edit it.
If the textile with the user name will be empty the prompt with the name field mandatory will be provided.

### Dev requirements and restrictions:
- Create textfield with name, surname, address, phone number.

## UI Mocks
You are free to choose your own design, but please follow the [App style guidelines](https://github.com/ios-course/ironfoudation-team-project/wiki/App-style-guidelines)

### PR DESCRIPTION
**Pull request by NadzeyaShpakouskaya**
## Summary
This PR is a part of the work to resolve #72.
Create the view that allows editing the information for the owner.

## UI Preview (Screenshots)

<details>
<summary>  iPhone 14 Pro Max - 6.7" </summary>
<p>

Screenshot 2023-05-30 at 09 54 43

</p>
</details>

<details>
<summary>  iPhone 14 Pro -  6.1" </summary>
<p>

Screenshot 2023-05-30 at 09 55 02

</p>
</details>

<details>
<summary>  iPhone SE 3rd gen - 4.7"</summary>
<p>

Screenshot 2023-05-30 at 09 54 52

</p>
</details>

## Changes
- Create the `EditOwnerInformationView`.
- Update `CHANGELOG.md` with the necessary information.

## Checks to complete
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] My code follows our style guide.
- [x] I've performed a self-review of my code.
- [x] PR's title has a conventional commit style (short and descriptive).
- [x] Every non-private interface is documented properly.
- [x] I've added tests for my code (if it makes sense).
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: Woof/Woof/View/OwnerProfileView/EditOwnerInformationView.swift [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    /// The phone of the owner
+    @Binding var phone: String
+
+    /// The address of the owner
+    @Binding var address: String
+
+    var body: some View {
+        VStack {
+            Text("Edit your information")
+            TextField("Name", text: $name)
```
Comment (iBYarchitect) [inline Woof/Woof/View/OwnerProfileView/EditOwnerInformationView.swift:20]: nit:
Is it possible to improve the user experience?
```suggestion
            TextField("Enter your name", text: $name)
```

**Review by iBYarchitect — APPROVED** [review]
Everything is fine, I just have one suggestion for improvement.

File: Woof/Woof/View/OwnerProfileView/EditOwnerInformationView.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        .padding()
+        .foregroundColor(.App.purpleDark)
+        .textFieldStyle(.roundedBorder)
+        .background(Color.App.purpleLight)
+        .cornerRadius(AppStyle.UIElementConstant.cornerRadius)
+        .frame(maxWidth: .infinity)
+    }
+}
+
+struct EditOwnerInformationView_Previews: PreviewProvider {
```
Comment (julyakmita) [inline Woof/Woof/View/OwnerProfileView/EditOwnerInformationView.swift:38]: Miss doc?

**Review by julyakmita — APPROVED** [review]
LGTM, I have only one nit.

File: Woof/Woof/View/OwnerProfileView/EditOwnerInformationView.swift [hunk 3]
```diff
@@ -0,0 +1,92 @@
+import SwiftUI
+
+/// A view to edit the owner information.
+struct EditOwnerInformationView: View {
+    /// The name of the owner
+    @Binding var name: String
+
+    /// The surname of the owner
```
[the remark this answers is not in the document]
Comment (NadzeyaShpakouskaya) [reply Woof/Woof/View/OwnerProfileView/EditOwnerInformationView.swift]: Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/77/commits/11ddea99623bf8ce08f31d1d106d2b08ee51c84b

**Pull request merged**